src/Shared/PackageManagers/MavenProjectManager.cs

```diff
@@ -27,8 +29,8 @@ public class MavenProjectManager : TypedManager<IManagerPackageVersionMetadata,
 
         public override string ManagerType => Type;
 
-        public const string DEFAULT_MAVEN_ENDPOINT = "https://repo1.maven.org/maven2";
-        public string ENV_MAVEN_ENDPOINT { get; set; } = DEFAULT_MAVEN_ENDPOINT;
+        public const MavenSupportedUpstream DEFAULT_MAVEN_ENDPOINT = MavenSupportedUpstream.MavenCentralRepository;
+        public MavenSupportedUpstream ENV_MAVEN_ENDPOINT { get; set; } = DEFAULT_MAVEN_ENDPOINT;
```

Comment:
I suspect this won't work with env variable overrides (https://github.com/microsoft/OSSGadget/blob/main/src/Shared/Helpers/EnvironmentHelper.cs#L17) -- we could update that function to account for non-string types, though.